lib/vanagon/platform.rb

```diff
@@ -237,8 +237,12 @@ def [](key)
     # @param target_repo [String] optional repo target for built packages defined
     #   at the project level
     # @return [String] relative path to where packages should be output to
-    def output_dir(target_repo = "")
-      @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
+    def output_dir(target_repo = nil)
+      if target_repo.nil?
+        @output_dir ||= "#{@os_name}-#{@os_version}-#{@architecture}"
+      else
+        @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
```

Comment:
No. If a project needs to continue building with the old paths, they simple have to specify `proj.target_repo='pc1'` in the project config. That will build and stage the artifacts in the exact same format we've always been building and staging projects. See the comment on this PR I made earlier today. If a user needed to further modify the directory built packages are staged in, then yes, they could set `output_dir`.